### PR TITLE
Feature/update schema

### PR DESCRIPTION
**Pull request by sshelomentsev**

### CODE REVIEW

File: scripts/generate.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ fi
 
 echo "VERSION: $VERSION"
 
-sed -i "s/projectVersion: .*/projectVersion: $VERSION/g" gradle.properties
-sed -i "s/^VERSION=.*/VERSION='$VERSION'/g" ./scripts/generate.sh
+sed -i '' -e "s/projectVersion = .*$/projectVersion = $VERSION/g" ./gradle.properties
```
Comment (ilfa) [inline scripts/generate.sh:22]: You just broke it for linux, try to check the platform like [here](https://github.com/fingerprintjs/fingerprint-pro-server-api-python-sdk/blob/main/generate.sh#L47)

File: scripts/sync.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ examplesList=(
 )
```
Comment (ilfa) [inline scripts/sync.sh:18]: Looks like you need to add new examples for new methods

File: sdk/src/main/java/com/fingerprint/api/FingerprintApi.java [hunk 3]
```diff
@@ -301,6 +304,88 @@ public ApiResponse<Response> getVisitsWithHttpInfo(String visitorId, String requ
...
+     <table summary="Response Details" border="1">
+       <tr><td> Status Code </td><td> Description </td><td> Response Headers </td></tr>
+       <tr><td> 200 </td><td> OK </td><td>  -  </td></tr>
+       <tr><td> 400 </td><td> Bad request </td><td>  -  </td></tr>
+       <tr><td> 403 </td><td> Forbidden </td><td>  -  </td></tr>
+       <tr><td> 404 </td><td> Not found </td><td>  -  </td></tr>
+       <tr><td> 409 </td><td> Conflict </td><td>  -  </td></tr>
+     </table>
+   */
+  public void updateEvent(String requestId, EventUpdateRequest eventUpdateRequest) throws ApiException {
```
Comment (ilfa) [inline sdk/src/main/java/com/fingerprint/api/FingerprintApi.java:323]: We need to add tests for this method

**Review by ilfa — CHANGES_REQUESTED** [review]
1. `update SDK according to new schema` not an informative message for release notes, we need to mention all the changes
2. We need to add tests and examples for new methods: `delete_visitor_data` and `update_event`

**Review by ilfa — APPROVED** [review]

**Review by ilfa — APPROVED** [review]

**Review by Orkuncakilkaya — APPROVED** [review]

**Pull request merged**